### PR TITLE
fix: move src path type 

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

#609 

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/chart/src/component/axisTitle.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import Component from './component';
 import { ChartState, Options } from '@t/store/store';
-import { AxisType } from '@src/component/axis';
 import { AxisTitleOption } from '@t/options';
 import { includes } from '@src/helpers/utils';
 import { FontTheme } from '@t/theme';
 import { getTitleFontString } from '@src/helpers/style';
 import { getAxisTheme } from '@src/helpers/axes';
 import { LabelModel } from '@t/components/axis';
+import { AxisType } from '@src/component/axis';
```
Comment (jungeun-cho) [inline apps/chart/src/component/axisTitle.ts:9]: 위치 변경한 이유가 있나요?

File: apps/chart/src/component/axisUsingCenterY.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import Component from './component';
 import Painter from '@src/painter';
-import { AxisType } from '@src/component/axis';
 import { ChartState, Options, CenterYAxisData, ViewAxisLabel } from '@t/store/store';
 import { makeTickPixelPositions, crispPixel } from '@src/helpers/calculator';
 import { TickModel, LineModel, AxisModels, LabelModel } from '@t/components/axis';
 import { AxisTheme } from '@t/theme';
 import { getAxisTheme } from '@src/helpers/axes';
 import { getTitleFontString } from '@src/helpers/style';
+import { AxisType } from '@src/component/axis';
```
Comment (jungeun-cho) [inline apps/chart/src/component/axisUsingCenterY.ts:9]: 여기도 위치를 변경한 이유가 있을까요?

**Review by jungeun-cho — APPROVED** [review]
확인했습니다. 수고하셨어요~!

File: apps/chart/src/component/axisUsingCenterY.ts (hunk 2 above)
Comment (jung-han) [reply apps/chart/src/component/axisUsingCenterY.ts:9]: 아 제가 enum을 선언 파일에 옮기면 안된다는것을 잊고...넣었다가 다시 옮겨서 위치가 변경이 되었습니다.

**Review by shiren — APPROVED** [review]
LGTM

**Pull request merged**